onmt/IO.py

```diff
@@ -236,6 +236,18 @@ def collect_features(fields):
         return feats
 
     @staticmethod
+    def collect_feature_dicts(fields):
```

Comment:
This is good. We'll fix this in a future PR (the torchtext people are working on it).